src/index.html

```diff
@@ -16,17 +17,24 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div data-qa="big">
+      <input
+        class="bar big-bar"
+        type="text"
+        data-qa="keypress"
```

Comment:
The `data-qa` attribute on this input element is set to `keypress`, but it should be set according to the task requirements. Please verify and update it accordingly.